dandi/tests/data/metadata/metadata2asset.json

```diff
@@ -1,5 +1,5 @@
 {
-  "identifier": "0b0a1a0b-e3ea-4cf6-be94-e02c830d54be",
+  "id": "dandiasset:0b0a1a0b-e3ea-4cf6-be94-e02c830d54be",
```

Comment:
hm, I thought that with the recent changes we still retain `identifier` and only provide `id` at the level of `dandi-api` queries (and may be only for json-ld serialization), thus minting it in addition/based on the `identifier`.  but even if "not only for json-ld" -- shouldn't original `identifier` also appear here?

again - one of my arguments was/is - with just an `id` users would need to "manually" split it inside of just using the value as `asset_id` for the API queries.  So by catering to the LD we are further sacrificing ease of use of this metadata for "more pragmatic" interactions ;-)

Reply:
i don't think this affects anything in the CLI now, because this is about conversion and the CLI explicitly ignores identifier.

Reply:
rright -- I think we are relying on API to provide us `asset_id`s in its listing of assets for a path, so indeed should not get broken. But my comment was not about breakage but rather about changing metadata so there is no 1-to-1 immediate correspondence between those records and API (`asset_id`).